README.rst

```diff
@@ -472,20 +472,30 @@ Then, open ``MatrixSDK.xcworkspace``.
 
 Tests
 =====
-The tests in the SDK Xcode project are both unit and integration tests. 
+The tests in the SDK Xcode project are both unit and integration tests.
 
 Unit tests classes use the suffix "UnitTests" to differentiate them. A unit test is a test that does not make any HTTP requests or uses mocked HTTP requests.
 
 Out of the box, the tests use one of the homeservers (located at
 http://localhost:8080) of the "Demo Federation of Homeservers"
 (https://matrix-org.github.io/synapse/develop/development/demo.html?highlight=demo#synapse-demo-setup).
 
+Before you install synapse you may need few dependencies to be installed on Mac OS:
+
+- **Homebrew**: run ``/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)”``. More information can be found here https://brew.sh
+- **python 3**: downloading the latest stable version should be fine. Download the ``.pkg`` and install it from here https://www.python.org/downloads/
+- **pipx**: with python installed run ``pip3 install --user pipx``
+- **Rust**: run ``curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh``. more information can be found here https://www.rust-lang.org/tools/install
+- **icu4c**: Run ``brew install icu4c``
+- **Update env variables for icu4c**: if you use zsh run ``echo 'export PATH="/opt/homebrew/opt/icu4c/bin:$PATH"' >> ~/.zshrc``. Otherwise try to update ``.bash_profile`` in the same way. If you can’t find ``icu4c`` in the default homebew folder try to run ``which icu4c`` to find the correct path and update the command above.
+- **pg_config**: you can get it by running ``brew install postgresql``
```

Comment:
In https://github.com/matrix-org/synapse/pull/13480/files, we postulated that libpq would be enough. Had you tried that out?

Reply:
(original issue: https://github.com/matrix-org/synapse/issues/13464)

Reply:
Do you mean `libpq` instead of `postgresql`?

Reply:
Yeah, exactly

Reply:
Hm, ok sorry I may have mixed this up then. It's odd though that postgres is not mentioned in the official install instructions?

https://matrix-org.github.io/synapse/latest/setup/installation.html